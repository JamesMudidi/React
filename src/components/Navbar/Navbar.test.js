import React from 'react';
import { shallow } from 'enzyme';
import Head from 'components/Navbar';

describe('Head', () => {
  it('matches the snapshot', () => {
    const component = shallow(<Head />);
    expect(component).toMatchSnapshot();
  });
});
