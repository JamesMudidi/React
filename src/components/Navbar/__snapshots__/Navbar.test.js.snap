// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head matches the snapshot 1`] = `
<Bootstrap(undefined)
  bg="info"
  collapseOnSelect={true}
  expand="lg"
  fixed="top"
  variant="dark"
>
  <NavbarBrand
    href="http://localhost:8080/"
  >
    iRepoter
  </NavbarBrand>
  <NavbarToggle
    aria-controls="responsive-navbar-nav"
    label="Toggle navigation"
  />
  <NavbarCollapse
    id="responsive-navbar-nav"
  >
    <Nav
      className="mr-auto"
      fill={false}
      justify={false}
    >
      <NavLink
        as={[Function]}
        disabled={false}
        href=""
      />
    </Nav>
    <Nav
      fill={false}
      justify={false}
    />
  </NavbarCollapse>
</Bootstrap(undefined)>
`;
